jaeger-client/build.gradle

```diff
@@ -7,3 +9,28 @@ dependencies {
 
     testCompile group: 'junit', name: 'junit', version: junitVersion
 }
+
+/* Shaded the same libraries as jaeger-thrift,
+ * leaving the slf4j dependency in place, as a *requirement*
+ * for a standalone fat jar. */
+shadowJar {
+    relocate 'com.google.gson'  , 'io.jaegertracing.vendor.com.google.gson'
+    relocate 'com.twitter'      , 'io.jaegertracing.vendor.com.twitter'
+    relocate 'okhttp'           , 'io.jaegertracing.vendor.okhttp'
+    relocate 'okio'             , 'io.jaegertracing.vendor.okio'
+    relocate 'org.apache'       , 'io.jaegertracing.vendor.org.apache'
+    relocate 'org.slf4j'        , 'io.jaegertracing.vendor.org.slf4j'
+    classifier 'bundle'
```

Comment:
Would prefer another name - `bundle` makes me think OSGi :smile: , but not sure what might be better. Could be something simple like `all` - but will let others comment.

Reply:
`all` sounds about right to me.